npm/private/test/snapshots/bzlmod/npm_defs.bzl

```diff
@@ -2121,7 +2163,6 @@ def npm_link_all_packages(name = "node_modules", imported_links = []):
             link_targets.append("//{}:{}/hello".format(bazel_package, name))
             link_503(name = "{}/handlebars-helpers/helper-date".format(name))
             link_targets.append("//{}:{}/handlebars-helpers/helper-date".format(bazel_package, name))
-            scope_targets["handlebars-helpers"] = scope_targets["handlebars-helpers"] + [link_targets[-1]] if "handlebars-helpers" in scope_targets else [link_targets[-1]]
```

Comment:
Note an actual change here. This package was declared in package.json as `"handlebars-helpers/helper-date": "file:...` or something like that... so it's not actually a scoped package but is an alias with a `/` in it.

I think the change here is correct to not treat it as a scoped package?